portfolio/src/main/webapp/script.js

```diff
@@ -442,15 +456,72 @@ function displayDirectionsAPI() {
   }
 }
 
-/*
-function repeatMarkers() {
-    fetch('/markers').then(response => response.json()).then((markers) => {
-    markers.forEach((marker) => {
-        var myPosition = new google.maps.LatLng(marker.lat, marker.lng);
-        if (google.maps.geometry.poly.isLocationOnEdge(myPosition, routeArray, 0.0064)) {
-            
-        }
-        });
+function repeatedMarkersUI() {
+    console.log("HEYYYYY");
+  document.getElementById('repeatedMarkerUI').style.display = "block";
+  setTimeout(hidePopup, 3000);
+}
+
+function hidePopup() {
+  console.log("@22222222222");
+  document.getElementById('repeatedMarkerUI').style.display = "none";
+  /*console.log("wait 3 minutes then show popup again");
+  setTimeout(popup, 3 * 60 * 1000);*/
+}
+
+
+function repeatMarkers(reportLat, reportLng, reportDate, reportTime, reportCrime) {
```

Comment:
1. This should be on the backend (in the servlet java code). 
2. The only thing that calls this is commented out so I don't think there's any checking of repeat crimes at all